docs/reference/glossary.asciidoc

```diff
@@ -66,7 +66,7 @@
   A filter is a non-scoring <<glossary-query,query>>, meaning that it does not score documents.
   It is only concerned about answering the question - "Does this document match?". 
   The answer is always a simple, binary yes or no. This kind of query is said to be made 
-  in a <<https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html,filter context>>, 
+  in a <<query-dsl/query_filter_context#,filter context>>, 
```

Comment:
This still gives an error when building the full reference docs (e.g. using `build_docs.pl --doc docs/reference/index.asciidoc --out ~/temp/asciidoc/  --open`):

```
/Users/xxx/temp/asciidoc/index.xml:78053: element link: validity error : Syntax of value for attribute linkend of link is not valid
  in a <link linkend="query-dsl/query_filter_context#">filter context</link>,
                                                      ^
/Users/xxx/temp/asciidoc/index.xml:78053: element link: validity error : IDREF attribute linkend references an unknown ID "query-dsl/query_filter_context#"
```
Which is why I was asking you earlier how you build the docs and what errors you get with the version I suggested (<<query-filter-context,filter context>>). Can you either tell me that so I can help or simply use that link instead? .